samples/net/openthread/coap_client/README.rst

```diff
@@ -1,14 +1,15 @@
 .. _coap_client_sample:
 
-openthread: coap_client
-##################
+OpenThread: CoAP Client
+#######################
 
-The openthread coap_client sample demonstrates interactions between nodes performing different Thread roles with the use of OpenThread and CoAP protocol.
+The OpenThreag CoAP Client sample demonstrates interactions between nodes performing different Thread roles with the use of OpenThread and CoAP protocol.
 
 Overview
 ********
 
-The client node can control the state of BSP_LED_3 on server nodes. It can turn on the LED either on every server node in the network with a multicast message, or on a single specific server node that is paired with the client node.
+The client node can control the state of LED 3 on server nodes. It can turn on the LED either on every server node in the network with a multicast message, or on a single specific server node that is paired with the client node.
```

Comment:
Also, can we have some more technical details here? Like what kind of messages it can send, what are message characteristics, when it can send it etc.?